### ISSUE TITLE
Not able to access SDCard on Chromebook.

### ISSUE THREAD

**Issue by liuchang0920** [termux/termux-app#506]
Thanks for this great product.

It seems that on chromebook, the SDCard is mounted in the path: ///_SDCard. And by executing "termux-setup-storage" still could not able to find the path to sdcard. 
Is this a constraint that chromeos forces? If so, this feature would only available when there is updates on chromeos then.

Thanks.

**Comment by liuchang0920** [comment]
I will be appreciated if there is any information on this

**Comment by Quasic** [comment]
I had to create the symlink manually. (```ln -s ...```) If that doesn't work, I'm afraid I don't have a chromebook to test.

**Comment by liuchang0920** [comment]
then i think it might need google to expose the visibility of sdcard

**Comment by sdrausty** [comment]
@liuchang0920 Are you familiar with https://termux.com/storage.html this information about setting up storage in Termux?  Ensure that storage permission is granted for the app.

**Comment by timnolte** [comment]
I believe Termux needs a code change as indicated on this [Chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=660189#c77) concerning how ARC has implemented the storage access API.

**Comment by Quasic** [comment]
@timnolte, that appears to be for Java only. Native binary applications wouldn't be able to use it without some kind of wrapper that runs Java code, which would probably be slow... That's why we don't use the normal access framework for Android.

**Comment by timnolte** [comment]
@quasic I'll be honest I've done zero Android development but I don't understand how your statement makes any sense. Just looking at the GitHub repo and the Termux source code is Java. Regardless the issue is that on a Chromebook SD Cards are accessed differently. The /sdcard location is not really and SD card storage location at all on a Chromebook.

**Comment by liuchang0920** [comment]
yes, i have tried according to the wiki page. it seems chromeos doesnt
expose sdcard as normal andoird device.

**Comment by Quasic** [comment]
@timnolte, the SDcard issue has been confusing since #20. Only recently could an app be written in non-Java. Very few of the commands in termux are written in Java, and these have to use the old dalvikvm, used by KitKat. The package sources can be found by following the packages repo.
